tests/e2e/default/sms/send-message.wdio-spec.js

```diff
@@ -48,7 +48,8 @@ describe('Send message', () => {
   before(async () => {
     await utils.saveDocs([...places.values(), bob]);
     await utils.createUsers([offlineUser]);
-    await loginPage.login(offlineUser);    
+    await utils.updateSettings({ district_admins_access_unallocated_messages: true }, true);
```

Comment:
Some of the sms messages that this file created ended unallocated. 
I considered changing the test and assert that feedback docs get generated. However, only the first test that created an unallocated record produced a feedback doc, and subsequent tests did not, due to us not generating feedback docs with same error message consecutively. 
This meant that test order was significant, as we would only assert feedback doc creation in one test.